challenge.js

```diff
@@ -9,7 +9,7 @@
  */
 
 // ANSWER
-
+I would go into the developer tools and use the console to alter the html.
```

Comment:
Yep, that'll work. Manipulating the DOM through JS is generally preferred, but altering HTML by hand is also effective. 3/4